Code/GraphMol/FMCS/testFMCS_Unit.cpp

```diff
@@ -2544,6 +2611,7 @@ int main(int argc, const char* argv[]) {
 
   T0 = nanoClock();
   t0 = nanoClock();
+  testBondStereo();
```

Comment:
You probably don't need to call this twice. I suspect that this instance is leftover from earlier debugging efforts.

Reply:
Yep, sorry.